src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -644,10 +651,37 @@ interface RouteData {
    */
   segments: { content: string; dynamic: boolean; spread: boolean; }[][];
   /**
-   * Function to render component in-place from a set of input data.
+	 * @param {any} data The optional parameters of the route
+	 * 
+	 * @description
+	 * A function that accepts a list of params, interpolates them with the route pattern, and returns the path name of the route.
+	 * 
+	 * ## Example
+	 * 
+	 * For a route such as `/blog/[...id].astro`, the `generate` function would return something like this:
+	 * 
+	 * ```js
+	 * console.log(generate({ id: 'presentation' })) // will log `/blog/presentation` 
+	 * ```
    * This is typically for internal use, so call with caution!
-   */
+	 */
   generate: (data?: any) => string;
+  /**
+	 * Whether the route is prerendered or not
+	 */
+	prerender: boolean;
+  /**
+	 * The paths of the physical files emitted by this route. When a route **isn't** prerendered, the value is either `undefined` or an empty array.
```

Comment:
Is "physical" the right adjective here, since files are not exactly concrete? If I forced you to choose another adjective, which one would you pick? :smile:

Reply:
Maybe we can rephrase it like this "The paths of the HTML files emitted by this route"